.evergreen/functions.yml

```diff
@@ -301,7 +301,12 @@ functions:
           set -e
           # Load environment variables
           eval $(.evergreen/print-compass-env.sh)
-
+          # The artifact file names are generated (in target.js) and added to the expansions
+          if [[ "${project}" == "10gen-compass-main" && "${branch_name}" == "main"  ]]; then
+            # In order to correclty use dev version identifer based on datetime, we will use
+            # evergreen's created_at expansion which is formatted as: 24_05_16_14_52_37.
+            export DEV_VERSION_IDENTIFIER=$(awk '{gsub("_", ""); print}' <<< ${created_at})
```

Comment:
NIT: should we put this code inside `.evergreen/print-compass-env.sh`?

Reply:
Also NIT: does it make sense to export the entire `DEV_VERSION=0.0.0-${somedatetime}` if we need to concatenate everywhere

Reply:
Yeah, i'd add it there and make concatenation part of it as well.